modules/controlfreec/1.1/controlfreec.smk

```diff
@@ -52,59 +52,72 @@ rule _get_map_refs:
                             "hg38": "unzip "}[w.genome_build],
         command2 = lambda w: {"grch37": " --wildcards --no-anchored 'out100m2*gem' && mv out100m2_hg19.gem ",
                             "hg19": " --wildcards --no-anchored 'out100m2*gem' && mv out100m2_hg19.gem ",
-                            "grch38": " > ",
-                            "hg38": " > "}[w.genome_build]
+                            "grch38": " -d ",
+                            "hg38": " -d "}[w.genome_build],
+        outdir = CFG["dirs"]["inputs"] + "references/{genome_build}/freec/"
     shell:
         "wget -O {output.tar} {params.url} "
-        "&& {params.command1} {output.tar} {params.command2} {output.gem}"
+        "&& {params.command1} {output.tar} {params.command2} {params.outdir}"
 
 # no chr for grch37 and grch38
 # chr for hg19 and hg38
-chromRegions = ("1", "2", "3", "4", "5", "6", "7", "8",
-        "9", "10", "11", "12", "13", "14", "15", "16",
-        "17", "18", "19", "20", "21", "22", "X", "Y")
+# Symlink chromosomes used (i.e. chr1-22,X,Y)
+checkpoint _controlfreec_input_chrs:
+    input:
+        chrs = reference_files("genomes/{genome_build}/genome_fasta/main_chromosomes_withY.txt")
+    output:
+        chrs = CFG["dirs"]["inputs"] + "references/{genome_build}/main_chromosomes_withY.txt"
+    run:
+        op.relative_symlink(input.chrs, output.chrs)
+
+
+def _controlfreec_get_chr_fastas(wildcards):
+    CFG = config["lcr-modules"]["controlfreec"]
+    chrs = checkpoints._controlfreec_input_chrs.get(**wildcards).output.chrs
+    with open(chrs) as file:
+        chrs = file.read().rstrip("\n").split("\n")
```

Comment:
You really should not clobber your chrs variable here. You're creating a new variable (chromosome) later. You should create it here instead.

Reply:
Updated to chromosome